activemerchant.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
   s.has_rdoc = true if Gem::VERSION < '1.7.0'
 
-  s.add_dependency('activesupport', '>= 3.2.14', '< 5.1')
+  s.add_dependency('activesupport', '>= 3.2.14', '< 5.2')
```

Comment:
:wave: :smile:  Hi Jason

I always feel there should be _some_ upper bound, just because there's no way to retroactively take it away in the future.

How about `'< 6'`? Or even better `'< 6.x'`, which will also forbid any `'6.0.0.alphas'`